### PR TITLE
Fixed plain text attribution typos for tools

### LINKED ISSUE #1155: Plain text attribution typos for tools
## Description

Anything marked with CC0 or PDM has two typos in the "Plain text" attribution tab:

1. Says "licensed" instead of "marked"
2. Says "to the terms" instead of "to view the terms"

You may view an adorable example here: https://search.creativecommons.org/photos/44874d17-45db-48d2-ab5d-fff03fb96a30

Screen Shot 2020-08-17 at 4 20 42 PM

## Resolution

- [x] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by avats-dev**
## Fixes

Fixes #1155 by @zackkrida

## Description

> Anything marked with **CC0** or **PDM** has two typos in the "Plain text" attribution tab:
> 1. Says "licensed" instead of "marked"
> 2. Says "to the terms" instead of "to view the terms"

Now, the fix resolves these issues as you can see in following screenshot:
Screenshot from 2020-08-18 11-13-24
### Successfully ran all the tests locally after editing:
Screenshot from 2020-08-18 11-15-03

### Previously:
image
### Now:
Screenshot from 2020-08-18 11-13-46

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ImageDetails/CopyLicense.vue [hunk 1]
```diff
@@ -113,9 +113,9 @@
         >
           {{ imageTitle }}
           <span v-if="image.creator"> by {{ image.creator }} </span>
-          is licensed under
+          is marked under
```
Comment (rochisha0) [inline src/components/ImageDetails/CopyLicense.vue:116]: I think you have to use `is marked under` for tools and `is licensed with` for licenses

Comment (zackkrida) [reply src/components/ImageDetails/CopyLicense.vue:116]: Yes @avats-dev, you need to check `isTool` here and use `marked under` for tools and `licensed with` for licenses.

**Comment by avats-dev** [comment]
> Thanks @zackkrida and @rochisha0 , for reviewing this and suggesting the needed updates.

I've updated the PR with the [changes](https://github.com/creativecommons/cccatalog-frontend/pull/1159/commits/cbc213adf7c5457c5b63a45795867f8973ae4695).

### Following screenshots visualize the changes tested locally.
**If there is a license**
Screenshot from 2020-08-19 00-09-30

**If there isn't**
Screenshot from 2020-08-19 00-10-12

**Comment by zackkrida** [comment]
Really appreciate your screenshots in your comments @avats-dev, very helpful.

**Comment by avats-dev** [comment]
Thanks @zackkrida for this subtle workflow, it was fun. Feel free to assign me something else, would love to contribute. Thanks for all the help. :rocket:

**Pull request merged**